luxonis_ml/data/datasets/luxonis_dataset.py

```diff
@@ -142,7 +142,7 @@ def _init_paths(self) -> None:
         self.media_path = self.local_path / "media"
         self.annotations_path = self.local_path / "annotations"
         self.metadata_path = self.local_path / "metadata"
-        self.masks_path = self.local_path / "masks"
+        self.arrays_path = self.local_path / "arrays"
```

Comment:
Is this a breaking change for datasets created earlier?

Reply:
No, the `masks_path` was a dead code.